src/components/Documentation/index.js

```diff
@@ -1,50 +1,24 @@
 /* global docsearch:readonly */
```

Comment:
why do we have it in some files?

Reply:
In this case, because we connect `docsearch` here https://github.com/iterative/dvc.org/pull/1025/files#diff-58826c9c508b82875d8f9a9872d19e2dR18 and it register global variable `window.docsearch`. Eslint didn't know about this global variable so we tell him about it here.

Reply:
👍